release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,83 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
 
 For more information about the Ironic provisioning service, see xref:../installing/installing_bare_metal_ipi/ipi-install-overview.adoc[Deploying installer-provisioned clusters on bare metal].
 
+[id="ocp-4-12-agent-based-install-asset-input"]
+==== Agent-based installation supports two input modes
+The Agent-based installation supports two input modes:
+
+**Preferred**
```

Comment:
Just a suggestion, but you could use block titles instead of bold for the lists here if you wanted. E.g.,

```
.Preferred
* `install-config.yaml` file
* `agent-config.yaml` file

.Optional
* Zero Touch Provisioning(ZTP) manifests

Reply:
Ye agreed I will change it to block titles